README.md

```diff
@@ -12,11 +12,11 @@ The backend of Amnesia is implemented in Java using the Spring framework. Its co
 
     To successfully build the Amnesia engine follow these steps:
     1.   Clone the project
-    2.   Download pixelmed.jar from http://www.dclunie.com/pixelmed/software/webstart/pixelmed.jar and install manaully to amnesia's Dependencies.
+    2.   Download pixelmed.jar from http://www.dclunie.com/pixelmed/software/webstart/pixelmed.jar and install manually to amnesia's Dependencies.
     3.   Go to controller/AppCon.java and set the variable "os" with desired operation system ("windows" or "linux")
     4.   Build the project
     5.   Take the .jar from the "scr/target" directory. 
-    6.   Downoload https://amnesia.openaire.eu/countryDistributionZip.ser and put it in the same folder where Amnesia's jar file will be executed. 
+    6.   Download https://amnesia.openaire.eu/countryDistributionZip.set and put it in the same folder where Amnesia's jar file will be executed. 
```

Comment:
The original .ser appears to be correct here; .set gives a 404!

Reply:
Oops! Will fix it.

Reply:
See #18.